### PR TITLE
Issue #4267: Fixed broken test

### PR DESCRIPTION
**Pull request by Kietzmann**
Related to #4267 
Placed InputEmptyDefault.java into the correct package and renamed it to fit requirements.

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/InputEmptyDefault.java [hunk 1]
```diff
@@ -1,89 +0,0 @@
-public class InputEmptyDefault {
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/InputEmptyDefault.java]: Why is this file deleted?
We expect file to be moved, with just changing package and changing class name.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockDefault.java [hunk 2]
```diff
@@ -1,87 +1,89 @@
...
-/**
- * Test case for detecting empty block statements.
- * @author Lars Kühne
- **/
-class InputEmptyBlockDefault
-{
-    static {
-        Boolean x = new Boolean(true);
+public class InputEmptyBlockDefault {
+    void method1(int a) {
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockDefault.java]: why are there so many changes in this file?
file should just be moved/copy. Changes should be minimal.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheckTest.java [hunk 3]
```diff
@@ -148,7 +148,7 @@ public void testInvalidOption() throws Exception {
         try {
             final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
 
-            verify(checkConfig, getPath("InputEmptyBlockDefault.java"), expected);
+            verify(checkConfig, getPath("InputEmptyBlockSemantic.java"), expected);
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheckTest.java]: this is probably reason for weird changes below.
You changed `InputEmptyBlockDefault` -> `InputEmptyBlockSemantic` and then changed `InputEmptyDefault` to same name `InputEmptyBlockDefault`.

Leave `InputEmptyBlockDefault` as it originally was.
Rename `InputEmptyDefault` to some other unique name.

**Comment by romani** [comment]
if commit cause a failure it is reverted ...... 

I did revert, issue was reopened - #4220 .
